.env.test

```diff
@@ -1,4 +1,4 @@
 BCRYPT_USER_SALT = '$2a$10$uXYG0R1562TiXBn0.X/nOu'
-KMS_USERNAME = ''
-KMS_PSWD = ''
+KMS_USERNAME = 'ibikecphsite'
+KMS_PSWD = 'b932bvjih56b2_D3'
```

Comment:
this should not have been pushed to the repo. it can be retrieved from the frontend anyways, but we should never store keys in the repo.